.travis.yml

```diff
@@ -9,15 +9,36 @@ env:
   global:
     # Travis limits maximum log size, we have to cut tests output
     - CSTOR_TEST_TRAVIS_LOG_MAX_LENGTH=800
+jobs:
+  include:
+    - os: linux
+      arch: amd64
+      env:
+        - RUN_UZFS_TESTS=1
```

Comment:
as the images are pushed finally from libcstor repo, its good to run kernel module build here at libcstor than the cstor repo

Reply:
I will say better to keep it in cstor repo only since cstor repo changes needs to be validated with kernel build.

Reply:
but finally build will be pushed from here right. So, even if change is made in cstor and there are no changes in libcstor, still, a build need to be triggered from libcstor to push the image.
So, as a final check, its better to run the kernel module at libcstor.

Reply:
otherwise, point is: changes in libcstor will not impact kernel build at all, as these are just user space ones and are not linked to kernel modules. If so, that seems to be fine.